dask/dataframe/utils.py

```diff
@@ -160,3 +160,7 @@ def get_categories(df):
     if iscategorical(df.index.dtype):
         result['.index'] = df.index.categories
     return result
+
+
+def is_frame_or_series(x):
+    return isinstance(x, (pd.Series, pd.DataFrame))
```

Comment:
Why not inline the `isinstance` check directly?

    isinstance(df, pd.core.generic.NDFrame) -> isinstance(df, (pd.Series, pd.DataFrame))